uint64 utime
uint8 num_joints
float64[] positions
geometry_msgs/Pose world2robot
